examples/plot_simulate_mpi_backend.py

```diff
@@ -0,0 +1,72 @@
+"""
+=======================================
+06. Use MPI backend for parallelization
+=======================================
+
+This example demonstrates how to use the MPI backend for
+simulating dipoles using HNN-core.
+
+The MPI backend allows running the simulation in parallel across neurons in the
+network even with a single trial. For this, you will
+need the :ref:`MPI related software <parallel>` installed. Note that if you
+want to simulate in parallel across trials, the Joblibs backend allows this
```

Comment:
```suggestion
want to simulate in parallel across trials, the Joblib backend allows this
```
consider adding the  :class:`~hnn_core.parallel_backends.Joblib` here too?